azurerm/resource_arm_container_group.go

```diff
@@ -599,18 +620,33 @@ func expandContainerGroupContainers(d *schema.ResourceData) (*[]containerinstanc
 	return &containers, &containerGroupPorts, &containerGroupVolumes
 }
 
-func expandContainerEnvironmentVariables(input interface{}) *[]containerinstance.EnvironmentVariable {
+func expandContainerEnvironmentVariables(input interface{}, secure bool) *[]containerinstance.EnvironmentVariable {
+
 	envVars := input.(map[string]interface{})
 	output := make([]containerinstance.EnvironmentVariable, 0)
 
-	for k, v := range envVars {
-		ev := containerinstance.EnvironmentVariable{
-			Name:  utils.String(k),
-			Value: utils.String(v.(string)),
+	if secure == true {
+
+		for k, v := range envVars {
+			ev := containerinstance.EnvironmentVariable{
+				Name:        utils.String(k),
+				SecureValue: utils.String(v.(string)),
+			}
+
+			output = append(output, ev)
 		}
-		output = append(output, ev)
-	}
 
+	} else {
```

Comment:
[https://github.com/Azure/azure-sdk-for-go/blob/v21.1.0/services/containerinstance/mgmt/2018-09-01/containerinstance/models.go#L643](https://github.com/Azure/azure-sdk-for-go/blob/v21.1.0/services/containerinstance/mgmt/2018-09-01/containerinstance/models.go#L643)

What's the relationship among Name, Value and SecureValue? What's difference between Value and SecureValue behind the scene?